### ISSUE TITLE
Coquille prescription 25

### ISSUE THREAD

**Issue by Geoscope19** [cnigfr/schema-plan-local-urbanisme#2]
Dans le doc "Standard CNIG PLU v2024", page 32, pour la prescription 25/00 il est indiqué L151-23 al. 2, c'est en fait **al.1**

**Comment by GT-CNIG-DDU** [comment]
Les éléments de continuités écologiques sont dans les deux paragraphes.
Je propose de conserver [L151-23](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006074075&idArticle=LEGIARTI000031211209) en supprimant **al.2** : 

![image](https://github.com/cnigfr/schema-plan-local-urbanisme/assets/79853274/a9ce01f8-ca3c-4c42-8a83-51faefb15dab)

**Comment by Geoscope19** [comment]
Oui, pourquoi pas, sachant toutefois que le premier paragraphe est codifié en 08/00

**Comment by GT-CNIG-DDU** [comment]
D'accord. Donc alinéa 1 : 

![image](https://github.com/cnigfr/schema-plan-local-urbanisme/assets/79853274/186e364b-2bf3-4479-bd7c-df57ce72cb81)
